File: backend/slotify/spring/src/main/java/com/slotify/backend/spring/service/dtos/ScheduleSummary.java
package com.slotify.backend.spring.service.dtos;

import lombok.Builder;
import lombok.Getter;
import lombok.ToString;

import java.time.LocalTime;
import java.util.UUID;

@Builder
@Getter
@ToString
public class ScheduleSummary {

    private UUID id;
    private Integer dayOfWeek;
    private LocalTime startTime;
    private LocalTime endTime;

}

File: frontend/app/src/main/java/com/raj/slotify/dtos/service/ScheduleSummary.kt
package com.raj.slotify.dtos.service

import com.google.gson.annotations.SerializedName
import java.time.LocalTime
import java.util.UUID

data class ScheduleSummary(

    @SerializedName("id")
    val id: UUID,

    @SerializedName("dayOfWeek")
    val dayOfWeek: Int,

    @SerializedName("startTime")
    val startTime: LocalTime,

    @SerializedName("endTime")
    val endTime: LocalTime

)
